dev-python/line_profiler/metadata.xml

```diff
@@ -9,6 +9,10 @@
     <email>sci@gentoo.org</email>
     <name>Gentoo Science Project</name>
   </maintainer>
+  <maintainer type="project">
+    <email>proxy-maint@gentoo.org</email>
+    <name>Gentoo Proxy Maintainers Project</name>
```

Comment:
A project indeed, so why do you call us 'herd' in the commit message? Do we look like sheep to you? ;-P

Reply:
Indeed. Fixed the commit message.